src/java.desktop/share/classes/sun/swing/SwingAccessor.java

```diff
@@ -316,6 +325,19 @@ public static void setAccessibleComponentAccessor(final AccessibleComponentAcces
         accessibleComponentAccessor = accessibleAccessor;
     }
 
+    public static AccessibleJTreeNodeCreateAccessor getAccessibleJTreeNodeCreateAccessor() {
+        var access = accessibleJTreeNodeCreateAccessor;
+        if (access == null) {
+            ensureClassInitialized(JTree.class);
```

Comment:
This probably doesn't work as JTree class initialization does not seem to trigger AccessibleJTree class initialization. The other one for AccessibleJTreeNode is dubious too, maybe it just happens that all accesses are correctly made only after the related classes are initialized.

Reply:
Therefore, in the swing accessor, class initialization is forced to be called, which initializes all nested classes. Look at the device of the Neighboring Accessories.

Reply:
This line calls the ```ensureClassInitialized()``` method, which performs a full initialization of the class... It calls: ```MethodHandles.lookup().ensureInitialized()```, which, as stated in the documentation , performs a full initialization.

Reply:
I know, but initializing a class does not necessarily initialize its nested/inner classes